core/src/com/unciv/Constants.kt

```diff
@@ -63,6 +63,9 @@ object Constants {
     const val unitActionParadrop = "Paradrop"
 
     const val futureTech = "Future Tech"
+    // Easter egg name. Hopefully is to hopefully avoid conflicts when later players can name their own religions.
+    // This religion name should never be displayed.
+    const val noReligionName = "The religion of TheLegend27" 
```

Comment:
From an _ad_???

Reply:
It was a huge meme at the time :p

Reply:
Ouch. Lemme check my birth year - oh my, my car's number plate says 1940, no wonder...